packages/apps/virtual-machine/values.yaml

```
@@ -0,0 +1,51 @@
+running: true
+
+## @param instanceType Virtual Machine instance type
+## @param instanceProfile Virtual Machine prefferences profile
+##
+instanceType: "u1.medium"
+instanceProfile: ubuntu
+
+## @param systemDisk.origin The HTTP URL for source image
+## @param systemDisk.storage The size of the disk allocated for the virtual machine
+## @param systemDisk.storageClass StorageClass used to store the data
+## 
```

yamllint

[error] trailing spaces

(trailing-spaces)